fisher.fish

```diff
@@ -1492,8 +1492,7 @@ function __fisher_key_bindings_remove -a plugin_name
 
     fish_indent < "$user_key_bindings" | command sed -n "/### $plugin_name ###/,/### $plugin_name ###/{s/^ *bind /bind -e /p;};" | builtin source ^ /dev/null
 
-    command sed "/### $plugin_name ###/,/### $plugin_name ###/d" < "$user_key_bindings" > "$user_key_bindings.$tmp"
-    command mv -f "$user_key_bindings.$tmp" "$user_key_bindings"
+    command sed --follow-symlinks -i "/### $plugin_name ###/,/### $plugin_name ###/d" "$user_key_bindings"
```

Comment:
@Perlence Okay, then, let's go with the alternative that does not break fisherman! 😄